src/module/actor/sheet/base.ts

```diff
@@ -323,13 +326,11 @@ abstract class ActorSheetPF2e<TActor extends ActorPF2e> extends ActorSheet<TActo
 
             input.addEventListener("blur", () => {
                 input.removeAttribute("name");
-                input.removeAttribute("style");
                 const propertyPath = input.dataset.property ?? "";
                 const preparedValue = getProperty(this.actor, propertyPath);
                 if (input instanceof HTMLInputElement) {
-                    const isModifier = input.classList.contains("modifier") && Number(preparedValue) >= 0;
-                    const value = isModifier ? `+${preparedValue}` : preparedValue;
-                    input.value = String(value);
+                    const isPositiveModifier = input.classList.contains("modifier") && Number(preparedValue) >= 0;
+                    input.value = isPositiveModifier ? `+${preparedValue}` : String(preparedValue);
```

Comment:
I think we have an addSign() util to handle this.

Reply:
PRs welcome

Reply:
Fine. NPC sheets are already too ugly.